apps/docs/src/pages/components/snackbar-provider.mdx

```diff
@@ -0,0 +1,186 @@
+
+```tsx live
+import {
+  HvButton,
+  HvOverflowTooltip,
+  HvSnackbarProvider,
+  useHvSnackbar,
+} from "@hitachivantara/uikit-react-core";
+
+export default function Demo() {
+  return (
+    <HvSnackbarProvider autoHideDuration={3000}>
+      <SnackbarButtons />
+    </HvSnackbarProvider>
+  );
+}
+
+const SnackbarButtons = () => {
+  const { enqueueSnackbar, closeSnackbar } = useHvSnackbar();
+
+  return (
+    <div className="flex gap-sm">
+      <HvButton
+        variant="secondarySubtle"
+        onClick={() => {
+          enqueueSnackbar("This is a success snackbar", { variant: "success" });
+        }}
+      >
+        Success
+      </HvButton>
+      <HvButton
+        variant="secondarySubtle"
+        onClick={() => {
+          enqueueSnackbar("This is a warning snackbar", { variant: "warning" });
+        }}
+      >
+        Warning
+      </HvButton>
+      <HvButton
+        variant="secondarySubtle"
+        onClick={() => {
+          enqueueSnackbar("This is an error snackbar", { variant: "error" });
+        }}
+      >
+        Error
+      </HvButton>
+      <HvButton
+        variant="secondarySubtle"
+        onClick={() => {
+          enqueueSnackbar("This is a default snackbar", { variant: "default" });
+        }}
+      >
+        Default
+      </HvButton>
+    </div>
+  );
+};
+```
+
+### Content
+
+Internally, the `HvSnackbarProvider` uses the `HvSnackbarContent` to render the snackbar. You can use the `snackbarContentProps` to customize the content of the snackbar.
+Check the [`HvSnackbar`](/components/snackbar) documentation for more information.
+
+```tsx live
+import {
+  HvButton,
+  HvOverflowTooltip,
+  HvSnackbarProvider,
+  useHvSnackbar,
+} from "@hitachivantara/uikit-react-core";
+
+export default function Demo() {
+  return (
+    <HvSnackbarProvider autoHideDuration={3000}>
+      <SnackbarButtons />
+    </HvSnackbarProvider>
+  );
+}
+
+const SnackbarButtons = () => {
+  const { enqueueSnackbar, closeSnackbar } = useHvSnackbar();
+
+  return (
+    <HvButton
+      variant="secondarySubtle"
+      onClick={() => {
+        const snackbarId = enqueueSnackbar("This is an action snackbar", {
+          snackbarContentProps: {
+            action: { id: "action", label: "Dismiss" },
+            onAction: (evt, action) => {
+              console.log("Clicked action", action);
+              closeSnackbar(snackbarId);
+            },
+            showIcon: true,
+            variant: "success",
+          },
+        });
+      }}
+    >
+      Action
+    </HvButton>
+  );
+};
+```
+
+### Label
+
+The snackbar label acceps a `React.ReactNode` which means you can pass any component as the label. This allows you to use the `HvOverflowTooltip` component to display a long message with ellipsis and a tooltip.
```

Comment:
I think we should reinforce/remind the user that snackbars shouldn't be more than 3 lines long. `HvOverflowTooltip` is a good workaround, but we could also suggest reducing/trimming the message size or using `HvBanner` instead

Reply:
I'll update the message.